### PR TITLE
Add iframe title attributes

### LINKED ISSUE #3961: iFrame title attributes
**Describe the bug**
We have two iframes - the outer iframe and the inner SMP iframe, we need to add title attributes to both these iframes to match the screen reader UX - This should be the same whether it's SMP on a live radio page, MAP or article page.

**Expected behaviour**
Outer iframe - title attribute 'Video player' if it's video content or 'Audio player' if it's audio content. If we cannot tell what the media type is, it should be 'Media player'. The title text should be in the local language.

Inner SMP iframe - title attribute '[headline]'

Note there should not be a page title element contained within either of the iframes. - covered in https://github.com/bbc/simorgh/issues/4231

We should carry out a developer accessibility swarm with all [supported AT](https://bbc.github.io/accessibility-news-and-you/accessibility-and-supported-assistive-technology), note that iframes are not acknowledges with all screen readers.

### PR DESCRIPTION
**Pull request by pharingee**
Resolves #3961

**Overall change:** 
Add title to MAP frames

**Code changes:**
- Update context translations
- Add title prop to media player components

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`) 
- [ ] This PR requires manual testing

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -70,7 +70,7 @@
     "@bbc/psammead-inline-link": "^1.3.11",
     "@bbc/psammead-locales": "^3.0.0",
     "@bbc/psammead-media-indicator": "^2.6.4",
-    "@bbc/psammead-media-player": "^1.1.1-alpha.7",
+    "@bbc/psammead-media-player": "^2.0.0",
```
Comment (HarveyPeachey) [inline package.json:73]: I thought the media player isn't a full release version yet so shouldn't it still be an alpha component?

File: src/app/containers/RadioPageBlocks/__snapshots__/index.test.jsx.snap [hunk 2]
```diff
@@ -121,6 +121,7 @@ exports[`Radio Page Blocks should match snapshot for AMP 1`] = `
               layout="fill"
               sandbox="allow-scripts allow-same-origin"
               src="https://www.test.bbc.co.uk/ws/av-embeds/media/bbc_amharic_radio/liveradio/am/amp"
+              title="Audio  player"
```
Comment (HarveyPeachey) [inline src/app/containers/RadioPageBlocks/__snapshots__/index.test.jsx.snap]: Looks like this snapshot has generated with an extra space between `audio` and `player`.

File: src/app/containers/RadioPageBlocks/__snapshots__/index.test.jsx.snap [hunk 3]
```diff
@@ -257,6 +258,7 @@ exports[`Radio Page Blocks should match snapshot for Canonical 1`] = `
           allowfullscreen=""
           class="c5"
           src="https://www.test.bbc.co.uk/ws/av-embeds/media/bbc_amharic_radio/liveradio/am"
+          title="Audio  player"
```
Comment (HarveyPeachey) [inline src/app/containers/RadioPageBlocks/__snapshots__/index.test.jsx.snap]: Same as comment above

File: src/app/containers/RadioPageBlocks/index.test.jsx [hunk 4]
```diff
@@ -11,7 +11,12 @@ import addIdsToBlocks from '../../routes/getInitialData/radioPage/addIdsToBlocks
 const pageData = addIdsToBlocks(amharicPageData);
 const blocks = path(['content', 'blocks'], pageData);
 
-const serviceContextMock = { service: 'amharic', script: latin, lang: 'am' };
+const serviceContextMock = {
+  service: 'amharic',
+  script: latin,
+  lang: 'am',
+  translations: { mediaAssetPage: { audioPlayer: 'Audio  player' } },
```
Comment (HarveyPeachey) [inline src/app/containers/RadioPageBlocks/index.test.jsx]:
```suggestion
  translations: { mediaAssetPage: { audioPlayer: 'Audio player' } },
```

File: src/app/lib/config/services/afaanoromoo.js [hunk 5]
```diff
@@ -49,6 +49,11 @@ export const service = {
       currentPage: 'Fuula kan ammaa',
       skipLinkText: 'Qabiyyeetti darbi',
       relatedContent: 'Odeessa kana irratti dabalata',
+      mediaAssetPage: {
```
Comment (HarveyPeachey) [inline src/app/lib/config/services/afaanoromoo.js:52]: Do we not know the translations for any services yet or are they all meant to use English?

**Review by HarveyPeachey — CHANGES_REQUESTED** [review]
Looks good so far just a few things

File: package.json (hunk 1 above)
Comment (pharingee) [reply package.json:73]: It is a full release now. We've fixed all the accessibility issues

File: src/app/lib/config/services/afaanoromoo.js (hunk 5 above)
Comment (pharingee) [reply src/app/lib/config/services/afaanoromoo.js:52]: We do not know the translations yet. The translations will come in later

Comment (pharingee) [reply src/app/lib/config/services/afaanoromoo.js:52]: We no don't know the translations yet. They are coming later

**Review by EinsteinNjoroge — APPROVED** [review]
Looks good 👍

**Review by ryanmccombe — APPROVED** [review]
👍

File: src/app/containers/MediaPlayer/index.jsx [hunk 6]
```diff
@@ -55,15 +55,21 @@ const MediaPlayerContainer = ({ blocks, placeholder }) => {
     isAmp,
     origin,
   });
+  const iframeTitle = translations.mediaAssetPage.mediaPlayer;
```
Comment (pjlee11) [inline src/app/containers/MediaPlayer/index.jsx:58]: This should probably use `path` from ramda

```suggestion
  const iframeTitle = path(['mediaAssetPage', 'mediaPlayer'], translations);
```

Comment (pjlee11) [reply src/app/containers/MediaPlayer/index.jsx:58]: Will also need to import `path` into this file

File: src/app/containers/RadioPageBlocks/Blocks/LiveRadio/index.jsx [hunk 7]
```diff
@@ -61,6 +61,7 @@ const LiveRadioContainer = ({ idAttr, externalId, id }) => {
           placeholderSrc={isAmp ? liveRadioPlaceholderImageSrc : null}
           showPlaceholder={!isAmp ? false : null}
           src={embedSource}
+          title={translations.mediaAssetPage.audioPlayer}
```
Comment (pjlee11) [inline src/app/containers/RadioPageBlocks/Blocks/LiveRadio/index.jsx:64]:
```suggestion
          path(['mediaAssetPage', 'audioPlayer'], translations);
```

And import path into this file

**Review by pjlee11 — CHANGES_REQUESTED** [review]
Few tweaks requested

**Comment by pharingee** [comment]
Closing due to commit signing issue:
Ref #4404

**Pull request closed without merging**